lms/static/js/Markdown.Converter.js

```diff
@@ -1285,27 +1285,12 @@ else
         //  attacklab: Utility functions
         //
 
-        var _problemUrlChars = /(?:["'*()[\]:]|~D)/g;

```

Comment:
I'd prefer us to upgrade the Markdown library instead of bringing in bits of a newer version.

Looking at the commit history of Markdown.Converter.js, we haven't made any modifications to this file since 7/25/2012 (we just moved it a couple of places). To be entirely confident that we haven't modified this file from its original form, it would be great to find the original someplace and diff the file.